src/main/sensors/voltage.c

```diff
@@ -84,8 +87,9 @@ const uint8_t supportedVoltageMeterCount = ARRAYLEN(voltageMeterIds);
 
 void voltageMeterReset(voltageMeter_t *meter)
 {
-    meter->filtered = 0;
+    meter->displayFiltered = 0;
     meter->unfiltered = 0;
+    meter->sagFiltered = 0;
```

Comment:
This needs to be bounded by `#ifdef USE_BATTERY_VOLTAGE_SAG_COMPENSATION` otherwise compilation fails if the feature is not enabled.

Reply:
We're still missing the needed `#ifdef USE_BATTERY_VOLTAGE_SAG_COMPENSATION` bounding here.